lib/capybara/webkit/driver.rb

```diff
@@ -359,7 +359,11 @@ def modal_action_options_for_browser(options)
     end
 
     def default_wait_time
-      Capybara.respond_to?(:default_max_wait_time) ? Capybara.default_max_wait_time : Capybara.default_wait_time
+      if respond_to?(:session_options) && session_options
+        session_options.default_max_wait_time
+      else
+        Capybara.respond_to?(:default_max_wait_time) ? Capybara.default_max_wait_time : Capybara.default_wait_time
```

Comment:
I wonder if we can axe some of this now too? We're only supporting capybara >= 2.3.0 at this point.